content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,76 @@
+
+### 📚 Library
+
+The library was [first implemented](https://github.com/excalidraw/excalidraw/pull/1787) by [petehunt](https://github.com/petehunt). Later after adding support for saving the library as a file, we eventually introduced a public directory for libraries. Visit [libraries.excalidraw.com](https://libraries.excalidraw.com) for more.
+
+### 📊 Excalicharts
+
+While we aim to keep Excalidraw simple to use, sometimes we hide little easter eggs that you need to find out for yourself (or follow our [Twitter account](https://twitter.com/excalidraw)). For example, you can copy any two column data from Excel, Spreadsheets, or comma separated values (CSV) from a text file and paste them into Excalidraw to get a nice chart out of it. The [first implementation](https://github.com/excalidraw/excalidraw/pull/1723) was done by [petehunt](https://github.com/petehunt) and the [improvements](https://github.com/excalidraw/excalidraw/pull/2495) by [lipis](https://github.com/lipis).
+
+### ✍️ Free Draw mode
+
+The [free hand draw](https://github.com/excalidraw/excalidraw/pull/1570) was added by [kbariotis](https://github.com/kbariotis).
+
+### 🏳 Translations
+
+From early on, users were asking for supporting multiple languages. The [final implementation](https://github.com/excalidraw/excalidraw/pull/638) of it was done by [vjeux](https://github.com/vjeux) and the automatic integration with our [Crowdin](https://crowdin.com/project/excalidraw) project was done by [lipis](https://github.com/lipis).
+
+### 💾 File system integration and file handling
+
+In Excalidraw, we use the [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) library to integrate with the file system of the operating system. This allows us to support a true open→edit→save workflow with proper over-saving and save-as on supporting browsers, and a fallback to downloading files and file uploads on other browsers. Read more about this feature in [tomayac](https://github.com/tomayac)'s [article](/browser-nativefs/). We also have [experimental support](https://web.dev/file-handling/#demo) for file type association, so that when you double-click an `.excalidraw` file in your file explorer, the Excalidraw PWA opens.
+
+### ⚙️ Gatsby plugin
+
+A plugin for Gatsby where you can very easily insert Excalidraw links and it will convert it to SVG during build time. We are actually using it in this post for our charts. Implemented by [trevorblades](https://github.com/trevorblades) and you can find it under [@excalidraw/gatsby-embedder-excalidraw](https://github.com/excalidraw/gatsby-embedder-excalidraw).
+
```

Comment:
```suggestion

# NPM package

One of the last things we've introduced this year was the completely rewritten npm package, available at [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw). Long time in the making (thanks to [@aakansha1216](https://twitter.com/aakansha1216) for most of the work), this package allows you to easily embed Excalidraw as a React component into your apps. We're nearing a 1.0.0 release, and will continue to actively develop it alongside our web app, so be sure to check it out and report any issues or requests to our [GitHub issue tracker](https://github.com/excalidraw/excalidraw/issues).

(As you may have noticed, the package is under [`@excalidraw`](https://www.npmjs.com/search?q=%40excalidraw) namespace under which we plan to also publish related packages. One of them is [`@excalidraw/utils`](https://www.npmjs.com/package/@excalidraw/utils).
```

Reply:
`#` is `H1` and we should have only one on the page.. but why not as part of the features?

Reply:
A mistake, it should be, of course.